entrypoint

```diff
@@ -5,24 +5,7 @@ set -e
 RUN_COMMAND="talisker.gunicorn.gevent webapp.app:app --bind $1 --worker-class gevent --name talisker-`hostname`"
 
 if [ "${FLASK_DEBUG}" = true ] || [ "${FLASK_DEBUG}" = 1 ]; then
```

Comment:
This needs to be an if else.

Reply:
Not sure I understand. Isn't this what we use for most of our websites? Fx:
https://github.com/canonical-web-and-design/jp.ubuntu.com/blob/master/entrypoint
https://github.com/canonical-web-and-design/charmed-kubeflow.io/blob/master/entrypoint

Reply:
NVM, I was looking at the diff and did not realise you swapped the logic inside the if to be an assignment.